dvc/command/get.py

```diff
@@ -55,4 +75,9 @@ def add_parser(subparsers, parent_parser):
     get_parser.add_argument(
         "--rev", nargs="?", help="Git revision (e.g. branch, tag, SHA)"
     )
+    get_parser.add_argument(
+        "--show-url",
+        action="store_true",
+        help="Returns path/url to the location in remote for specified path",
```

Comment:
Could use the description from https://github.com/iterative/dvc/pull/3130/files .

Reply:
Though, looks like it is still up for discussion, so i'm sure we will modify it while working on the docs. Let's keep it as is for now 👍

Reply:
Ah yeah I would change this description, its confusing. Sorry, didn't notice before merge. Maybe
"Returns URL to the storage location of the target data".

Reply:
Maybe "Returns URL to the storage location of the target data."
See https://github.com/iterative/dvc.org/pull/936#discussion_r368746593